src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  max-width: $cal-max-width;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100px;
+    width: $calendar-width;
+    border: 1px solid black;
+    font-size: 30px;
+    cursor: pointer;
+    background-color: #eee;
+    transition: 0.5s;
```

Comment:
Always add a transition for styles that you want smoothly change

Reply:
Good suggestion. Fully implemented